### DISCUSSION TITLE
server list doesn't have the motd columb in the table, only users and the cuntry

### DISCUSSION THREAD

**Discussion by peter1384** [BearWare/TeamTalk5#2331] [Ideas]
after all of the pull requests and with currently very up to date master, everything is fine, but we only have name, users and cuntry columbs, we don't have the motd columb. please add it.

**Comment by Nardol** [comment]
Having a **column** containing a text which can have more than one line is a very strange idea in my opinion.

**Comment by peter1384** [comment]
we've had it before, I don't know how it has been disappeared
@Nardol yes we can have it, because all of the text is just being red by the columb.

**Reply by CoBC** [reply]
This MOTD column was never be present

**Comment by peter1384** [comment]
@CoBC and it's not going to be added? or it will be?
